force-app/main/default/lwc/geBatchGiftEntryHeader/__tests__/geBatchGiftEntryHeader.test.js

```diff
@@ -136,6 +136,9 @@ describe('c-ge-batch-gift-entry-header', () => {
 
             expect(handler).toHaveBeenCalled();
             expect(handler.mock.calls[0][0].type).toBe('processbatch');
+            expect(buttons[0].disabled).toBe(true);
```

Comment:
I was debating either putting this assertion in a new test, or asserting that `isGiftBatchProcessing` is set to true instead of asserting that the buttons got disabled as I'm not sure if the current implementation follows good unit testing practice since the disabled functionality is already technically being tested, any feedback is much appreciated!

Reply:
I think this works great, no need for a separate test since you're just adding additional checks to the same process.